server/rest/src/test/scala/org/infinispan/rest/IntegrationTest.scala

```diff
@@ -689,6 +689,8 @@ class IntegrationTest extends RestServerTestBase {
          eTagFirst.equals(eTagSecond))
    }
 
+   // This test is *DISABLED* because of ISPN-5992
+   @Test(enabled = false)
```

Comment:
We usually put the JIRA reference in the `description` attribute of the `@Test` annotation, no *emphasis* required :)